Add non-blocking dpg/pog labels to release summary

iarspider opened the pull request:
closes cms-sw/cms-bot#1944

cmsbuild commented:
A new Pull Request was created by  @iarspider  for branch master.

@cmsbuild, @smuzaffar, @aandvalenzuela, @iarspider can you please review it and eventually sign? Thanks.
@perrotta, @dpiparo, @rappoccio you are the release manager for this.
cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

iarspider commented:
Diff from running `python release-notes -n CMSSW_13_0_0_pre4 CMSSW_13_1_0_pre1 REL/CMSSW_13_0_0_pre4/el8_amd64_gcc11 REL/CMSSW_13_1_0_pre1/el8_amd64_gcc11` on master and on this branch:
```diff
-- #40880 from @suchandradutta: Fix to a customize python script in Phase2 Tracker Digizer   `simulation` `upgrade`
+- #40880 from @suchandradutta: Fix to a customize python script in Phase2 Tracker Digizer   `simulation` `upgrade` `trk`
-- #40873 from @arossi83: Fix Tracks related Pixel quantities at HLT DQM `dqm` `reconstruction`
-- #40869 from @AdrianoDee: Fix `cudaMemcpyAsync` for `localCoordToHostAsync` `reconstruction` `heterogeneous`
+- #40873 from @arossi83: Fix Tracks related Pixel quantities at HLT DQM `dqm` `reconstruction` `tracking` `trk`
+- #40869 from @AdrianoDee: Fix `cudaMemcpyAsync` for `localCoordToHostAsync` `reconstruction` `heterogeneous` `tracking`
-- #40863 from @mmusich: `SiPixelDynamicInefficiency_PayloadInspector`: add region descriptor to title of inefficiency parametrization plots  `db`
+- #40863 from @mmusich: `SiPixelDynamicInefficiency_PayloadInspector`: add region descriptor to title of inefficiency parametrization plots  `db` `trk`
-- #40856 from @mmusich: `SiPixelDynamicInefficiency_PayloadInspector`: add polynomial representation plot of inefficiency vs inst lumi `db`
+- #40856 from @mmusich: `SiPixelDynamicInefficiency_PayloadInspector`: add polynomial representation plot of inefficiency vs inst lumi `db` `trk`
-- #40835 from @mmusich: Disentangle `TrackerTrackHitFilter` from phase-0 Strip conditions and activate common alignment track refitter sequence for phase-2 `alca` `reconstruction`
-- #40832 from @fwyzard: CUDA, ROCm and Alpaka-related updates `alca` `operations` `reconstruction` `upgrade` `heterogeneous`
+- #40835 from @mmusich: Disentangle `TrackerTrackHitFilter` from phase-0 Strip conditions and activate common alignment track refitter sequence for phase-2 `alca` `reconstruction` `tracking` `trk`
+- #40832 from @fwyzard: CUDA, ROCm and Alpaka-related updates `alca` `operations` `reconstruction` `upgrade` `heterogeneous` `tracking` `trk`
-- #40812 from @Dr15Jones: Changed binary to logical operations in TrackingTools packages `reconstruction`
-- #40810 from @Dr15Jones: Changed binary to logical operation in OptimalHelixPlaneCrossing `reconstruction`
+- #40812 from @Dr15Jones: Changed binary to logical operations in TrackingTools packages `reconstruction` `tracking`
+- #40810 from @Dr15Jones: Changed binary to logical operation in OptimalHelixPlaneCrossing `reconstruction` `tracking`
-- #40805 from @Dr15Jones: Removed unused variables in SiPixelChargeReweightingAlgorithm `simulation`
+- #40805 from @Dr15Jones: Removed unused variables in SiPixelChargeReweightingAlgorithm `simulation` `trk`
-- #40800 from @Dr15Jones: Remove unused variables from RecoVertex/BeamSpotProducer `alca` `reconstruction`
+- #40800 from @Dr15Jones: Remove unused variables from RecoVertex/BeamSpotProducer `alca` `reconstruction` `tracking`
-- #40793 from @mmusich: `usesResource` in `ConditionDBWriter` `alca` `core` `db`
+- #40793 from @mmusich: `usesResource` in `ConditionDBWriter` `alca` `core` `db` `trk`
-- #40773 from @elfontan: uGT fix for muon showers full emulation and for HMT mismatches in the DQM workflow `dqm` `l1`
+- #40773 from @elfontan: uGT fix for muon showers full emulation and for HMT mismatches in the DQM workflow `dqm` `l1` `l1t`
-- #40763 from @CTPPS: PPS diamond mapping XML for 2023 run `alca` `reconstruction`
-- #40762 from @jshin96: [PileupJetId, Puppi] Pileup ID input variable fix, puppi weight ValueMap access, optional photon protection for existing puppi weights `reconstruction` `xpog`
+- #40763 from @CTPPS: PPS diamond mapping XML for 2023 run `alca` `reconstruction` `ctpps`
+- #40762 from @jshin96: [PileupJetId, Puppi] Pileup ID input variable fix, puppi weight ValueMap access, optional photon protection for existing puppi weights `reconstruction` `xpog` `jetmet`
-- #40757 from @Dr15Jones: Only compare geometry in Geometry/TrackerGoeometryBuilder test `geometry`
-- #40754 from @mmusich: Merge SiStrip Hit resolution code in cmssw  `alca`
+- #40757 from @Dr15Jones: Only compare geometry in Geometry/TrackerGoeometryBuilder test `geometry` `trk`
+- #40754 from @mmusich: Merge SiStrip Hit resolution code in cmssw  `alca` `trk`
-- #40751 from @hftsoi: HCAL Fb DQM monitoring at layer1 `dqm`
+- #40751 from @hftsoi: HCAL Fb DQM monitoring at layer1 `dqm` `hcal`
-- #40746 from @mmusich: [TkAl all-in-one tool] Improve documentation and unit test coverage `alca`
-- #40744 from @dzuolo: Adding a check on the BS transverse widths in OnlineBeamSpotESProducer `alca` `reconstruction`
+- #40746 from @mmusich: [TkAl all-in-one tool] Improve documentation and unit test coverage `alca` `trk`
+- #40744 from @dzuolo: Adding a check on the BS transverse widths in OnlineBeamSpotESProducer `alca` `reconstruction` `tracking`
-- #40742 from @adewit: Phase2 tracker: Bug-fix for clashes in TBPX L2 in T25 and T32 `geometry`
+- #40742 from @adewit: Phase2 tracker: Bug-fix for clashes in TBPX L2 in T25 and T32 `geometry` `trk`
-- #40732 from @zhokin2: HCAL: bug fixed in DPGAnalysis/HcalTools for 2023 Run3 `dqm`
-- #40727 from @mmusich: add a unit Test for `OnlineBeamMonitor` `dqm` `db`
+- #40732 from @zhokin2: HCAL: bug fixed in DPGAnalysis/HcalTools for 2023 Run3 `dqm` `hcal`
+- #40727 from @mmusich: add a unit Test for `OnlineBeamMonitor` `dqm` `db` `tracking`
-- #40710 from @laurenhay: Run3JetID `operations` `reconstruction` `xpog`
+- #40710 from @laurenhay: Run3JetID `operations` `reconstruction` `xpog` `jetmet`
-- #40675 from @dan131riley: Use an indexed sort for expensive comparisons in MuonShowerInformationFiller `reconstruction`
+- #40675 from @dan131riley: Use an indexed sort for expensive comparisons in MuonShowerInformationFiller `reconstruction` `muon`
-- #40634 from @VinInn: Spedup HCAL Digitization `alca` `simulation` `db`
-- #40618 from @VinInn: Speedup strip digitization `reconstruction` `simulation`expected minor regressions from fix of ZeroSuppression algorithm
+- #40634 from @VinInn: Spedup HCAL Digitization `alca` `simulation` `db` `hcal`
+- #40618 from @VinInn: Speedup strip digitization `reconstruction` `simulation` `trk`expected minor regressions from fix of ZeroSuppression algorithm
```

iarspider commented:
The [HTML notes](https://cmssdt.cern.ch/SDT/ReleaseNotes/CMSSW_13/CMSSW_13_0_0_pre4.html) are generated from release notes, so this should work for both.

smuzaffar commented:
+externals

looks good

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next master IBs after it passes the integration tests. This pull request will now be reviewed by the release team before it's merged. @perrotta, @dpiparo, @rappoccio (and backports should be raised in the release meeting by the corresponding L2)
